lib/doorkeeper/models/access_token_mixin.rb

```diff
@@ -10,6 +10,12 @@ module AccessTokenMixin
     include ActiveModel::MassAssignmentSecurity if defined?(::ProtectedAttributes)
 
     included do
+      unless respond_to?(:ordered_by)
```

Comment:
Also can be removed, because `BaseRecord` implements this method, so it is a dead code. Let's ApplicationController do his work with deprecation :)